lk2nd/device/dts/msm8952/rules.mk

```diff
@@ -19,4 +20,3 @@ ADTBS += \
 
 DTBS += \
     $(LOCAL_DIR)/msm8917-huawei-agassi.dtb \
-
```

Comment:
I'd appreciate if you can drop this hunk from your diff, I'd prefer to avoid unnecessary churn of empty lines.

Reply:
That was a mistake, fixed.